src/api.ts

```diff
@@ -26,3 +26,13 @@ function get<T>(url: string): Promise<T> {
 export const getTodos = () => get<Todo[]>('/todos');
```

Comment:
Consider renaming the function to follow the naming conventions for methods. A more descriptive name like 'fetchTodos' would be clearer.